test/bdd/features/kms_api.feature

```diff
@@ -10,6 +10,12 @@ Feature: KMS operations
 
   Scenario: User creates a key
     Given Key Server is running on "localhost" port "8070"
-      And User has created a keystore on the server
+      And User has created an empty keystore on the server
```

Comment:
what's the use case for this?

Reply:
creating a keystore doesn't automatically creates a key, that test was to emphasize that "initial" state

Reply:
I was just wondering on the use cases for allowing empty key stores to be created in the first place